db/mongo.go

```diff
@@ -0,0 +1,87 @@
+package db
+
+import (
+	"log"
+
+	"gopkg.in/mgo.v2"
+	"gopkg.in/mgo.v2/bson"
+)
+
+//Technology collects the slack user ID of people who use or want to learn a
+//given technology, identified by Name
+type Technology struct {
+	ID       string   `json:"id"`
+	Name     string   `json:"name"`
+	Users    []string `json:"users"`
+	Learners []string `json:"learners"`
+}
+
+func getSession() *mgo.Session {
+	session, err := mgo.Dial("mongodb://localhost")
```

Comment:
Would that mean I also wouldn't need the 'defer session.Close()' ? Was thinking of abstracting the session/collection  3 lines, maybe just establish the collection and cache that and never worry about the session itself?